source/_docs/z-wave/controllers.markdown

```diff
@@ -73,32 +73,32 @@ Firmware can be upgraded using the below directions:
   - Z-Wave.Me Razberry 2 (500 series)
   - Z-Wave.Me Razberry 1 (300 series)
```

Comment:
I cannot select the headline, but if we're throwing USB sticks and GPIO modules under the generic term "adapter" now, should we then merge the Raspberry pi modules with the corresponding sections, since they are technically also 500/700/800 series?